utilities/gsheets-import/resultsImporter.py

```diff
@@ -21,21 +21,27 @@
 
 from oauth2client.service_account import ServiceAccountCredentials
 
-GOOGLE_SHEET_URL = ""
-
 SCOPES = ['https://spreadsheets.google.com/feeds',
           'https://www.googleapis.com/auth/drive']
 
+# TEST_STATES = ["Pass", "Warning", "Fail", "Manual", "Not Applicable", "Not Implemented", "Test Disabled",
+#                "Could Not Test"]
+TEST_STATES = ["Pass", "Fail", "Warning", "Not Implemented", "Test Disabled", "Could Not Test", "Manual",
+               "Not Applicable"]
+
 
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("--json", required=True)
+    parser.add_argument("--sheet", required=True)
+    parser.add_argument("--credentials", required=True)
+    parser.add_argument("--start_col", type=int)
```

Comment:
Looks like this needs a default argument, otherwise it results in:
```
Traceback (most recent call last):
  File "resultsImporter.py", line 134, in <module>
    main()
  File "resultsImporter.py", line 67, in main
    start_col = 1+reserved_cols
TypeError: unsupported operand type(s) for +: 'int' and 'NoneType'
```

Reply:
Ah, OK. I'd default to zero (or, hmm, should this be one-indexed like `gsheet` itself). Or would you prefer 4 as the JT-NM version?

Reply:
Zero (or one) is fine by me. Probably best to keep this more widely applicable and we can adjust how we call it for specific events.